src/Logging/Logging/LogOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using CodeMonkeys.Configuration;
+
+namespace CodeMonkeys.Logging
+{
+    public abstract class LogOptions : Options
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Convert this 'abstract' class to a concrete class with a protected constructor.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)